core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -3045,10 +3061,7 @@ protected function save($varValue)
 			$this->arrSubmit[$this->strField] = $varValue;
 			$this->varValue = StringUtil::deserialize($varValue);
 
-			if (\is_object($this->objActiveRecord))
```

Comment:
I don't think we can change this, `onsubmit_callback` might access the `activeRecord` and expect the new values to be here.

Reply:
We have to, we need the old data.